docs/DeveloperGuide.md

```diff
@@ -707,3 +707,214 @@ Coming soon
 ### Saving data
 
 Coming soon
+
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
```

Comment:
currently, our DG has the sections 
* Appendix: Requirements
* Appendix: Instructions for manual testing
* Appendix: Planned Enhancements

should we rename them to Appendix A: Requirements, Appendix B... etc?